### ISSUE TITLE
Any prerequisites for AirTest Version 0.1.1- AirtestIDE_2018-06-19_py3_win64 as it crashes as soon as we clicks on Exe file

### ISSUE THREAD

**Issue by SUDARSHANCHAUDHARI** [AirtestProject/Airtest#85]
Hello
I just download the new version of AirTest IDE V 0.1.1.
I want to know, is there any prerequisites requires to run this version as last version V0.0.18 running smoothly on my machine but not this one I tried 64 bit.

**Comment by yimelia** [comment]
hi~@SUDARSHANCHAUDHARI 

just to confirm: is your installation path in plain english? does your installation path have any special characters?

**Comment by Meteorix** [comment]
Maybe you can try 32bit version

**Comment by SUDARSHANCHAUDHARI** [comment]
there is some firewall problem with Artest IDE.exe.
I am unable to unzip the 32-bit version on my machine.
It showing the message about unsafe exe file and blocking the unzip process

**Comment by Meteorix** [comment]
we have released a new version, please try the latest version

**Comment by SUDARSHANCHAUDHARI** [comment]
Yes it is working the latest version (AirtestIDE_2018-06-19_py3_win64.zip)
thank you!!
